<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Data • TmS</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Data">
<meta property="og:description" content="TmS">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TmS</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.0.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/TmS.html">Tutorial</a>
</li>
<li>
  <a href="../articles/data.html">Data</a>
</li>
<li>
  <a href="../articles/shinyapp.html">TmS Correlates Exploration</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/wwylab/TmS">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="data_files/header-attrs-2.14.1/header-attrs.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Data</h1>
            
            <h4 class="date">4/26/2022</h4>
      
      
      <div class="hidden name"><code>data.Rmd</code></div>

    </div>

    
    
<p>Here, we provide the download links of estimated tumor-specific total RNA expression proportion (<span class="math inline">\(\pi\)</span>), TmS, and intrinsic signature genes for each cancer type from TCGA, EOPC, METABRIC and TRACERx cohorts, as described in our paper [1].</p>
<p>The links in the <strong>TmS summary</strong> column provide text files named <code>TmS summary.csv</code>; each file includes the following columns for the corresponding cancer type: - Sample ID - <span class="math inline">\(\pi\)</span> estimate: estimated tumor-specific total RNA expression proportion - (1-<span class="math inline">\(\rho\)</span>)/(<span class="math inline">\(\rho\)</span>*<span class="math inline">\(\psi\)</span>): for TCGA, this is derived by consensus <span class="math inline">\(TmS / [2\pi/(1-\pi)]\)</span>. See <a href="TmS.html">Tutorial page</a> for details about <strong>Consensus TmS estimation</strong> - TmS</p>
<p>The links in the <strong>Intrinsic tumor signature genes</strong> column provide text files named <code>Intrinsic_signature_genes.txt</code>; each includes the signature genes that are ranked by the gene selection scores for the corresponding cancer type [2, 3].</p>
<table class="table">
<thead><tr>
<th>
Cohort
</th>
<th>
Cancer type
</th>
<th>
TmS summary
</th>
<th>
Intrinsic tumor signature genes
</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="15">
TCGA
</td>
<td>
BLCA
</td>
<td>
<a href="./etc/TmS/TCGA/BLCA/BLCA_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/BLCA/BLCA_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
BRCA
</td>
<td>
<a href="./etc/TmS/TCGA/BRCA/BRCA_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/BRCA/BRCA_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
CRC
</td>
<td>
<a href="./etc/TmS/TCGA/CRC/CRC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/CRC/CRC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
HNSC
</td>
<td>
<a href="./etc/TmS/TCGA/HNSC/HNSC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/HNSC/HNSC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
KICH
</td>
<td>
<a href="./etc/TmS/TCGA/KICH/KICH_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/KICH/KICH_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
KIRC
</td>
<td>
<a href="./etc/TmS/TCGA/KIRC/KIRC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/KIRC/KIRC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
KIRP
</td>
<td>
<a href="./etc/TmS/TCGA/KIRP/KIRP_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/KIRP/KIRP_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
LIHC
</td>
<td>
<a href="./etc/TmS/TCGA/LIHC/LIHC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/LIHC/LIHC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
LUAD
</td>
<td>
<a href="./etc/TmS/TCGA/LUAD/LUAD_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/LUAD/LUAD_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
LUSC
</td>
<td>
<a href="./etc/TmS/TCGA/LUSC/LUSC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/LUSC/LUSC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
PAAD
</td>
<td>
<a href="./etc/TmS/TCGA/PAAD/PAAD_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/PAAD/PAAD_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
PRAD
</td>
<td>
<a href="./etc/TmS/TCGA/PRAD/PRAD_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/PRAD/PRAD_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
STAD
</td>
<td>
<a href="./etc/TmS/TCGA/STAD/STAD_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/STAD/STAD_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
THCA
</td>
<td>
<a href="./etc/TmS/TCGA/THCA/THCA_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/THCA/THCA_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
UCEC
</td>
<td>
<a href="./etc/TmS/TCGA/UCEC/UCEC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TCGA/UCEC/UCEC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
EOPC
</td>
<td>
</td>
<td>
<a href="./etc/TmS/EOPC/EOPC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/EOPC/EOPC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>
<tr>
<td>
METABRIC
</td>
<td>
</td>
<td>
<a href="./etc/TmS/METABRIC/METABRIC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/METABRIC/METABRIC_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>

<tr>
<td>
TRACERx
</td>
<td>
</td>
<td>
<a href="./etc/TmS/TRACERx/TRACERx_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/TRACERx/TRACERx_Intrinsic_signature_genes.txt">link</a>
</td>
</tr>


<tr>
<td>
SCAN-B
</td>
<td>
TNBC
</td>
<td>
<a href="./etc/TmS/SCAN-B/SCAN_B_TNBC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/SCAN-B/SCAN_B_TNBC_Intrinsic_genes_symbol.txt">link</a>
</td>
</tr>


<tr>
<td>
FUSCC
</td>
<td>
TNBC
</td>
<td>
<a href="./etc/TmS/FUSCC/FUSCC_TNBC_TmS_summary.csv">link</a>
</td>
<td>
<a href="./etc/TmS/FUSCC/FUSCC_TNBC_Intrinsic_genes_symbol.txt">link</a>
</td>
</tr>


</tbody>
</table>
<p><strong>Note</strong>: EOPC, Early-Onset Prostate Cancer [4]; METABRIC, Molecular Taxonomy of Breast Cancer International Consortium [5]; TRACERx, Tracking Cancer Evolution through therapy [6].</p>
<div id="reference" class="section level2">
<h2 class="hasAnchor">
<a href="#reference" class="anchor"></a>Reference</h2>
<p>[1] Cao, S. et al. Estimation of tumor cell total mRNA expression in 15 cancer types predicts disease progression. Nat Biotechnol (2022). https://doi.org/10.1038/s41587-022-01342-x.</p>
<p>[2] Ahn, J. et al. DeMix: Deconvolution for mixed cancer transcriptomes using raw measured data. Bioinformatics 29, 1865–1871 (2013).</p>
<p>[3] Wang, Z. et al. Transcriptome Deconvolution of Heterogeneous Tumor Samples with Immune Infiltration. iScience 9, 451–460 (2018).</p>
<p>[4] Gerhauser, C. et al. Molecular Evolution of Early-Onset Prostate Cancer Identifies Molecular Risk Markers and Clinical Trajectories. Cancer Cell 34, 996–1011 (2018).</p>
<p>[5] Curtis, C. et al. The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. Nature 486, 346–352 (2012).</p>
<p>[6] Jamal-Hanjani, M. et al. Tracking the evolution of non-small-cell lung cancer. N. Engl. J. Med. 376, 2109–2121 (2017).</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Shuangxi Ji.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
